src/Styles/Mixin.js

```diff
@@ -0,0 +1,9 @@
+import { css } from "styled-components";
+const Mixin = {
+  center: (position) => css`
+  display: flex;
+  justify-content: ${position};
+  align-items: center;
+  `,
+};
+export default Mixin;
```

Comment:
```suggestion
import { css } from "styled-components";
const mixin = {
  center: (position) => css`
    display: flex;
    justify-content: ${position};
    align-items: center;
  `,
};

export default mixin;
```
- 구분되는 요소 간 한줄 씩 띄워주시고 들여쓰기도 잘 써주세요
- 변수명 카멜케이스로 써주세요. 소문자로 시작해야겠죠?

Reply:
넵 수정했습니다!